Classes/CloudFlareAPI.cs

```diff
@@ -21,221 +21,357 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
  */
+
 using System;
-using System.Text;
+using System.Collections.Generic;
 using System.IO;
 using System.Net;
+using System.Net.Sockets;
+using System.Text;
 using System.Web.Script.Serialization;
+using CloudFlareDDNS.Classes.JsonObjects.Cloudflare;
 
 namespace CloudFlareDDNS
 {
     /// <summary>
     /// Provides functions for interfacing with the CloudFlare REST API
     /// </summary>
-    class CloudFlareAPI
+    internal class CloudFlareAPI
     {
-
-
+        JavaScriptSerializer jss = new JavaScriptSerializer();
         /// <summary>
         /// Logic to update records
+        /// And return changes
         /// </summary>
-        public void updateRecords(JsonResponse fetchedRecords)
+        public List<Result> updateRecords(Get_dns_records_response fetchedRecords)

```

Comment:
Class names should not contain underscores, please rename to GetDnsRecordsResponse